src/components/Home/BlogContent/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import Parser, { Item } from 'rss-parser';
+
+import Post from './Post';
+
+import * as Inject from '../../../stores/MenuStateStore';
+
+import './styles.css';
+
+interface BCState {
+	data: any;
+}
+
+interface BCProps {}
+
+@inject((stores: Inject.Props) => ({
+	blogState: stores.store.blogState,
+}))
+@observer
+class BlogContent extends Component<BCProps, BCState> {
+	constructor(props: BCProps) {
+		super(props);
+
+		this.state = {
+			data: [],
+		};
+
+		const parser = new Parser({
+			defaultRSS: 4,
+			timeout: 1500,
+		});
+		const CORSFree = 'https://cors-anywhere.herokuapp.com/';
+
+		parser.parseURL(CORSFree + 'https://blog.neonkid.xyz/rss').then(feed => {
+			this.setState({ data: feed.items });
+		});
```

Comment:
CORS Free recommend to use only development mode...
Not recommended Production deploy..